services/templates.js

```diff
@@ -32,7 +36,7 @@ function create(data) {
 
 }
 
-function edit(templateID, data) {
+function editTemplate(templateID, data) {
     const {name = "", creator = "", html = ""} = data;
```

Comment:
please use Linter in your project. [read more about ESlint](https://eslint.org/)